dipy/reconst/quick_squash.pyx

```diff
@@ -7,6 +7,8 @@ cimport cython
 
 import numpy as np
 
+from dipy.testing.decorators import warning_for_keywords
```

Comment:
can you remove this import ?

Reply:
I have no idea how this even got there, but it's gone as of fbe63ba66